pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl

```diff
@@ -136,6 +140,7 @@ task CramToBam {
     File cram_file
     String output_basename
     Int disk_size
+    Int memory = 8
```

Comment:
Going from 7.5 to 8 pushes this to the next larger class of machine (although it's possible that's changed under PAPIv2?).  Probably better to make it 7

Reply:
Good point. I have changed this and the others as well